addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -577,6 +561,76 @@ async function toInline($editable, cssRules, $iframe) {
             }
         }
     }
+    return displaysToRestore;
+}
```

Comment:
I trust that all these new functions only contain code that was moved from their original location without changes. If that is not the case please let me know so that I make a more thorough checking.